enacts/wat_bal/maproom_monit.py

```diff
@@ -492,23 +553,45 @@ def wat_bal_plots(
             error_msg="Please ensure all input boxes are filled for the calculation to run."
         )
 
+    #Save actual start of ts2
+    ts2_start = ts2["T"][0]
+
+    #Find corresponding day closer to ts
     p_d2 = calc.sel_day_and_month(
         precip["T"], int(planting2_day), calc.strftimeb2int(planting2_month)
```

Comment:
Why are we passing around a string representation of the month, and converting it to an integer each time we need to use it in a calculation? Wouldn't it be more straightforward to use integers for values (not labels) in the menu? (Not a blocker.)

Reply:
This is a legacy of controls.py's Month function. I can make an issue to revisit this as we see fit